src/string/isomorphism.rs

```diff
@@ -0,0 +1,59 @@
+/*
+Given two strings s and t, determine whether they are isomorphic.
+The two strings are isomorphic if the characters in s can be replaced by some mapping relation to get t
+*/
+use std::collections::HashMap;
+
```

Comment:
Add method docstring here
```suggestion

/// Determines whether two strings are isomorphic.
///
/// Two strings are considered isomorphic if the characters in one string can be replaced
/// by some mapping relation to obtain the other string.
///
/// # Arguments
///
/// * `s` - The first string.
/// * `t` - The second string.
///
/// # Returns
///
/// `true` if the strings are isomorphic, `false` otherwise.
```

Reply:
Yeah, sorry for the confusion, just remove duplicated lines.